experimenter/experimenter/experiments/models.py

```diff
@@ -439,6 +439,29 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         default=list,
     )
     tags = models.ManyToManyField(Tag, blank=True, related_name="experiments")
+    qa_run_date = models.DateField("QA Run Date", blank=True, null=True, default=None)
```

Comment:
I don't think we want both blank and null as valid values for these fields. I believe Django forms work better with blank=true and a default value of "" instead of trying to make null the "empty" value.